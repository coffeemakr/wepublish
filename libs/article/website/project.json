{
  "name": "article-website",
  "$schema": "../../../node_modules/nx/schemas/project-schema.json",
  "sourceRoot": "libs/article/website/src",
  "projectType": "library",
  "tags": [],
  "targets": {
    "lint": {
      "executor": "@nx/linter:eslint",
      "outputs": ["{options.outputFile}"],
      "options": {
        "lintFilePatterns": ["libs/article/website/**/*.{ts,tsx,js,jsx}"]
      }
    },
    "build": {
      "executor": "@nx/js:tsc",
      "outputs": ["{options.outputPath}"],
      "options": {
        "outputPath": "dist/libs/article/website",
        "tsConfig": "libs/article/website/tsconfig.lib.json",
        "packageJson": "libs/article/website/package.json",
        "main": "libs/article/website/src/index.ts",
        "assets": ["libs/article/website/*.md"],
        "updateBuildableProjectDepsInPackageJson": true
      }
    },
    "test": {
      "executor": "@nx/jest:jest",
      "outputs": ["{workspaceRoot}/coverage/{projectRoot}"],
      "options": {
        "jestConfig": "libs/article/website/jest.config.ts",
        "passWithNoTests": true
      }
    }
  }
}
