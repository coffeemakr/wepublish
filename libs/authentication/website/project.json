{
  "name": "authentication-website",
  "$schema": "../../../node_modules/nx/schemas/project-schema.json",
  "sourceRoot": "libs/authentication/website/src",
  "projectType": "library",
  "tags": [],
  "targets": {
    "lint": {
      "executor": "@nx/linter:eslint",
      "outputs": ["{options.outputFile}"],
      "options": {
        "lintFilePatterns": ["libs/authentication/website/**/*.{ts,tsx,js,jsx}"]
      }
    },
    "build": {
      "executor": "@nx/js:tsc",
      "outputs": ["{options.outputPath}"],
      "options": {
        "outputPath": "dist/libs/authentication/website",
        "tsConfig": "libs/authentication/website/tsconfig.lib.json",
        "packageJson": "libs/authentication/website/package.json",
        "main": "libs/authentication/website/src/index.ts",
        "assets": ["libs/authentication/website/*.md"],
        "updateBuildableProjectDepsInPackageJson": true
      }
    },
    "test": {
      "executor": "@nx/jest:jest",
      "outputs": ["{workspaceRoot}/coverage/{projectRoot}"],
      "options": {
        "jestConfig": "libs/authentication/website/jest.config.ts",
        "passWithNoTests": true
      }
    }
  }
}
