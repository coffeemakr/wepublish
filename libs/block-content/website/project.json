{
  "name": "block-content-website",
  "$schema": "../../../node_modules/nx/schemas/project-schema.json",
  "sourceRoot": "libs/block-content/website/src",
  "projectType": "library",
  "tags": [],
  "targets": {
    "lint": {
      "executor": "@nx/linter:eslint",
      "outputs": ["{options.outputFile}"],
      "options": {
        "lintFilePatterns": ["libs/block-content/website/**/*.{ts,tsx,js,jsx}"]
      }
    },
    "build": {
      "executor": "@nx/js:tsc",
      "outputs": ["{options.outputPath}"],
      "options": {
        "outputPath": "dist/libs/block-content/website",
        "tsConfig": "libs/block-content/website/tsconfig.lib.json",
        "packageJson": "libs/block-content/website/package.json",
        "main": "libs/block-content/website/src/index.ts",
        "assets": ["libs/block-content/website/*.md"],
        "updateBuildableProjectDepsInPackageJson": true
      }
    },
    "test": {
      "executor": "@nx/jest:jest",
      "outputs": ["{workspaceRoot}/coverage/{projectRoot}"],
      "options": {
        "jestConfig": "libs/block-content/website/jest.config.ts",
        "passWithNoTests": true
      }
    }
  }
}
