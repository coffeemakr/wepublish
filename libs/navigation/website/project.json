{
  "name": "navigation-website",
  "$schema": "../../../node_modules/nx/schemas/project-schema.json",
  "sourceRoot": "libs/navigation/website/src",
  "projectType": "library",
  "tags": [],
  "targets": {
    "lint": {
      "executor": "@nx/linter:eslint",
      "outputs": ["{options.outputFile}"],
      "options": {
        "lintFilePatterns": ["libs/navigation/website/**/*.{ts,tsx,js,jsx}"]
      }
    },
    "build": {
      "executor": "@nx/js:tsc",
      "outputs": ["{options.outputPath}"],
      "options": {
        "outputPath": "dist/libs/navigation/website",
        "tsConfig": "libs/navigation/website/tsconfig.lib.json",
        "packageJson": "libs/navigation/website/package.json",
        "main": "libs/navigation/website/src/index.ts",
        "assets": ["libs/navigation/website/*.md"],
        "updateBuildableProjectDepsInPackageJson": true
      }
    },
    "test": {
      "executor": "@nx/jest:jest",
      "outputs": ["{workspaceRoot}/coverage/{projectRoot}"],
      "options": {
        "jestConfig": "libs/navigation/website/jest.config.ts",
        "passWithNoTests": true
      }
    }
  }
}
