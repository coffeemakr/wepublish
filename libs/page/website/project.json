{
  "name": "page-website",
  "$schema": "../../../node_modules/nx/schemas/project-schema.json",
  "sourceRoot": "libs/page/website/src",
  "projectType": "library",
  "tags": [],
  "targets": {
    "lint": {
      "executor": "@nx/linter:eslint",
      "outputs": ["{options.outputFile}"],
      "options": {
        "lintFilePatterns": ["libs/page/website/**/*.{ts,tsx,js,jsx}"]
      }
    },
    "build": {
      "executor": "@nx/js:tsc",
      "outputs": ["{options.outputPath}"],
      "options": {
        "outputPath": "dist/libs/page/website",
        "tsConfig": "libs/page/website/tsconfig.lib.json",
        "packageJson": "libs/page/website/package.json",
        "main": "libs/page/website/src/index.ts",
        "assets": ["libs/page/website/*.md"],
        "updateBuildableProjectDepsInPackageJson": true
      }
    },
    "test": {
      "executor": "@nx/jest:jest",
      "outputs": ["{workspaceRoot}/coverage/{projectRoot}"],
      "options": {
        "jestConfig": "libs/page/website/jest.config.ts",
        "passWithNoTests": true
      }
    }
  }
}
