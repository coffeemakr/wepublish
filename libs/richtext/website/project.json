{
  "name": "richtext-website",
  "$schema": "../../../node_modules/nx/schemas/project-schema.json",
  "sourceRoot": "libs/richtext/website/src",
  "projectType": "library",
  "tags": [],
  "targets": {
    "lint": {
      "executor": "@nx/linter:eslint",
      "outputs": ["{options.outputFile}"],
      "options": {
        "lintFilePatterns": ["libs/richtext/website/**/*.{ts,tsx,js,jsx}"]
      }
    },
    "build": {
      "executor": "@nx/js:tsc",
      "outputs": ["{options.outputPath}"],
      "options": {
        "outputPath": "dist/libs/richtext/website",
        "tsConfig": "libs/richtext/website/tsconfig.lib.json",
        "packageJson": "libs/richtext/website/package.json",
        "main": "libs/richtext/website/src/index.ts",
        "assets": ["libs/richtext/website/*.md"],
        "updateBuildableProjectDepsInPackageJson": true
      }
    },
    "test": {
      "executor": "@nx/jest:jest",
      "outputs": ["{workspaceRoot}/coverage/{projectRoot}"],
      "options": {
        "jestConfig": "libs/richtext/website/jest.config.ts",
        "passWithNoTests": true
      }
    }
  }
}
