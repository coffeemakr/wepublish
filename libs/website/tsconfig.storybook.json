{
  "extends": "./tsconfig.json",
  "compilerOptions": {
    "emitDecoratorMetadata": true,
    "outDir": ""
  },
  "files": [
    "../../node_modules/@nx/react/typings/styled-jsx.d.ts",
    "../../node_modules/@nx/react/typings/cssmodule.d.ts",
    "../../node_modules/@nx/react/typings/image.d.ts"
  ],
  "exclude": ["src/**/*.spec.ts", "src/**/*.spec.js", "src/**/*.spec.tsx", "src/**/*.spec.jsx"],
  "include": [
    "../**/website/src/**/*.stories.ts",
    "../**/website/src/**/*.stories.js",
    "../**/website/src/**/*.stories.jsx",
    "../**/website/src/**/*.stories.tsx",
    "../**/website/src/**/*.stories.mdx",
    ".storybook/*.js",
    "preview.tsx",
    "main.ts"
  ]
}
