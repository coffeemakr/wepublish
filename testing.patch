diff --git a/libs/api/src/lib/db/article.ts b/libs/api/src/lib/db/article.ts
index 7359af6a..ea27be09 100644
--- a/libs/api/src/lib/db/article.ts
+++ b/libs/api/src/lib/db/article.ts
@@ -14,11 +14,11 @@ export interface ArticleData {
   readonly lead?: string | null
   readonly seoTitle?: string | null
   readonly slug?: string | null
-  readonly tags: string[]
+  readonly tags?: string[]
 
   readonly canonicalUrl?: string | null
 
-  readonly properties: MetadataProperty[]
+  readonly properties?: MetadataProperty[]
 
   readonly imageID?: string | null
   readonly authors: ArticleRevisionAuthor[]
@@ -107,7 +107,7 @@ export type ArticleRevisionWithRelations = PrismaArticleRevision & {
 
 export type ArticleWithRevisions = PrismaArticle & {
   draft: ArticleRevisionWithRelations | null
-  pending: ArticleRevisionWithRelations | null
+  pending?: ArticleRevisionWithRelations | null
   published: ArticleRevisionWithRelations | null
 }
 
diff --git a/libs/api/src/lib/db/page.ts b/libs/api/src/lib/db/page.ts
index fb73c289..6f0ed1b5 100644
--- a/libs/api/src/lib/db/page.ts
+++ b/libs/api/src/lib/db/page.ts
@@ -14,9 +14,9 @@ export interface PageData {
 
   readonly title: string
   readonly description?: string | null
-  readonly tags: string[]
+  readonly tags?: string[]
 
-  readonly properties: MetadataProperty[]
+  readonly properties?: MetadataProperty[]
 
   readonly imageID?: string | null
 
@@ -85,7 +85,7 @@ export type PageRevisionWithProperties = PrismaPageRevision & {
 
 export type PageWithRevisions = PrismaPage & {
   draft: PageRevisionWithProperties | null
-  pending: PageRevisionWithProperties | null
+  pending?: PageRevisionWithProperties | null
   published: PageRevisionWithProperties | null
 }
 
diff --git a/libs/api/src/lib/graphql/article.ts b/libs/api/src/lib/graphql/article.ts
index 8d80fb27..77357e01 100644
--- a/libs/api/src/lib/graphql/article.ts
+++ b/libs/api/src/lib/graphql/article.ts
@@ -275,7 +275,7 @@ export const GraphQLPublicArticle: GraphQLObjectType<PublicArticle, Context> =
         type: GraphQLNonNull(GraphQLList(GraphQLNonNull(GraphQLMetadataPropertyPublic))),
         resolve: ({properties}) => {
           return properties
-            .filter(property => property.public)
+            ?.filter(property => property.public)
             .map(({key, value}) => ({key, value}))
         }
       },
diff --git a/libs/api/src/lib/graphql/article/article.private-mutation.ts b/libs/api/src/lib/graphql/article/article.private-mutation.ts
index 43a6a3ee..049c7e54 100644
--- a/libs/api/src/lib/graphql/article/article.private-mutation.ts
+++ b/libs/api/src/lib/graphql/article/article.private-mutation.ts
@@ -178,7 +178,8 @@ export const duplicateArticle = async (
       createMany: {
         data: socialMediaAuthors.map(({authorId}) => ({authorId}))
       }
-    }
+    },
+    blocks: []
   }
 
   return articleClient.create({
@@ -303,7 +304,8 @@ export const unpublishArticle = async (
               createMany: {
                 data: socialMediaAuthors.map(({authorId}) => ({authorId}))
               }
-            }
+            },
+            blocks: []
           },
           update: {
             ...revision,
@@ -333,7 +335,8 @@ export const unpublishArticle = async (
               createMany: {
                 data: socialMediaAuthors.map(({authorId}) => ({authorId}))
               }
-            }
+            },
+            blocks: []
           }
         }
       },
@@ -491,7 +494,8 @@ export const publishArticle = async (
                 createMany: {
                   data: socialMediaAuthors.map(({authorId}) => ({authorId}))
                 }
-              }
+              },
+              blocks: []
             },
             update: {
               ...revision,
@@ -521,7 +525,8 @@ export const publishArticle = async (
                 createMany: {
                   data: socialMediaAuthors.map(({authorId}) => ({authorId}))
                 }
-              }
+              },
+              blocks: []
             }
           }
         },
@@ -579,7 +584,8 @@ export const publishArticle = async (
               createMany: {
                 data: socialMediaAuthors.map(({authorId}) => ({authorId}))
               }
-            }
+            },
+            blocks: []
           },
           update: {
             ...revision,
@@ -609,7 +615,8 @@ export const publishArticle = async (
               createMany: {
                 data: socialMediaAuthors.map(({authorId}) => ({authorId}))
               }
-            }
+            },
+            blocks: []
           }
         }
       },
diff --git a/libs/api/src/lib/graphql/page.ts b/libs/api/src/lib/graphql/page.ts
index 56d380f0..6b63518e 100644
--- a/libs/api/src/lib/graphql/page.ts
+++ b/libs/api/src/lib/graphql/page.ts
@@ -210,7 +210,7 @@ export const GraphQLPublicPage = new GraphQLObjectType<PublicPage, Context>({
     properties: {
       type: GraphQLNonNull(GraphQLList(GraphQLNonNull(GraphQLMetadataPropertyPublic))),
       resolve: ({properties}) => {
-        return properties.filter(property => property.public).map(({key, value}) => ({key, value}))
+        return properties?.filter(property => property.public).map(({key, value}) => ({key, value}))
       }
     },
 
diff --git a/libs/api/src/lib/graphql/page/page.private-mutation.ts b/libs/api/src/lib/graphql/page/page.private-mutation.ts
index 8d21a995..456df84e 100644
--- a/libs/api/src/lib/graphql/page/page.private-mutation.ts
+++ b/libs/api/src/lib/graphql/page/page.private-mutation.ts
@@ -140,7 +140,8 @@ export const duplicatePage = async (
       createMany: {
         data: duplicatedProperties
       }
-    }
+    },
+    blocks: []
   }
 
   return pageClient.create({
@@ -236,7 +237,8 @@ export const unpublishPage = async (
               createMany: {
                 data: properties
               }
-            }
+            },
+            blocks: []
           },
           update: {
             ...revision,
@@ -250,7 +252,8 @@ export const unpublishPage = async (
               createMany: {
                 data: properties
               }
-            }
+            },
+            blocks: []
           }
         }
       },
@@ -387,7 +390,8 @@ export const publishPage = async (
                 createMany: {
                   data: properties
                 }
-              }
+              },
+              blocks: []
             },
             update: {
               ...revision,
@@ -401,7 +405,8 @@ export const publishPage = async (
                 createMany: {
                   data: properties
                 }
-              }
+              },
+              blocks: []
             }
           }
         },
@@ -443,7 +448,8 @@ export const publishPage = async (
               createMany: {
                 data: properties
               }
-            }
+            },
+            blocks: []
           },
           update: {
             ...revision,
@@ -454,7 +460,8 @@ export const publishPage = async (
               createMany: {
                 data: properties
               }
-            }
+            },
+            blocks: []
           }
         }
       },
