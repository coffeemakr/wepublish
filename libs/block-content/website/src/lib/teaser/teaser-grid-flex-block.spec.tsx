import {runStorybookTests} from '@wepublish/testing/website'
import * as stories from './teaser-grid-flex-block.stories'

describe('Teaser Grid Flex Block', () => {
  runStorybookTests(stories)
})
