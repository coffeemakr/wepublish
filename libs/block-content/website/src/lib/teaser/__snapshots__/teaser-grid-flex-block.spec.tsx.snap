// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Teaser Grid Flex Block should render Default 1`] = `
<DocumentFragment>
  .emotion-0 {
  display: grid;
  -webkit-column-gap: 16px;
  column-gap: 16px;
  row-gap: 40px;
  grid-template-columns: 1fr;
  -webkit-align-items: stretch;
  -webkit-box-align: stretch;
  -ms-flex-align: stretch;
  align-items: stretch;
}

@media (min-width:600px) {
  .emotion-0 {
    grid-template-columns: 1fr 1fr;
  }
}

@media (min-width:900px) {
  .emotion-0 {
    grid-template-columns: repeat(12, 1fr);
  }
}

.emotion-2 {
  display: grid;
}

@media (min-width:900px) {
  .emotion-2 {
    grid-column-start: 1;
    grid-column-end: 4;
    grid-row-start: 1;
    grid-row-end: 5;
  }
}

.emotion-4 {
  margin: 0;
  font: inherit;
  color: inherit;
  -webkit-text-decoration: none;
  text-decoration: none;
  color: inherit;
  display: grid;
}

.emotion-5 {
  display: grid;
  gap: 8px;
  grid-template-rows: auto;
  grid-auto-rows: max-content;
}

.emotion-7 {
  max-width: 100%;
  width: 100%;
  height: auto;
  aspect-ratio: auto 1;
  height: 100%;
  width: 100%;
  object-fit: cover;
}

.emotion-11 {
  margin: 0;
  font-family: "Roboto","Helvetica","Arial",sans-serif;
  font-weight: 400;
  font-size: 1.5rem;
  line-height: 1.334;
  letter-spacing: 0em;
}

.emotion-12 {
  margin: 0;
  font-family: "Roboto","Helvetica","Arial",sans-serif;
  font-weight: 500;
  font-size: 1.25rem;
  line-height: 1.6;
  letter-spacing: 0.0075em;
}

.emotion-13 {
  display: grid;
}

@media (min-width:900px) {
  .emotion-13 {
    grid-column-start: 4;
    grid-column-end: 8;
    grid-row-start: 1;
    grid-row-end: 7;
  }
}

.emotion-26 {
  display: grid;
}

@media (min-width:900px) {
  .emotion-26 {
    grid-column-start: 8;
    grid-column-end: 13;
    grid-row-start: 1;
    grid-row-end: 9;
  }
}

.emotion-37 {
  display: grid;
}

@media (min-width:900px) {
  .emotion-37 {
    grid-column-start: 1;
    grid-column-end: 4;
    grid-row-start: 5;
    grid-row-end: 9;
  }
}

.emotion-50 {
  display: grid;
}

@media (min-width:900px) {
  .emotion-50 {
    grid-column-start: 4;
    grid-column-end: 8;
    grid-row-start: 7;
    grid-row-end: 13;
  }
}

.emotion-61 {
  display: grid;
}

@media (min-width:900px) {
  .emotion-61 {
    grid-column-start: 8;
    grid-column-end: 13;
    grid-row-start: 9;
    grid-row-end: 17;
  }
}

<div
    class="emotion-0 emotion-1"
  >
    <article
      class="emotion-2 emotion-3"
      x="0"
      y="0"
    >
      <a
        class="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineNone emotion-4"
        href="https://example.com"
      >
        <div
          class="emotion-5 emotion-6"
        >
          <img
            alt="An image description"
            class="emotion-7 emotion-8"
            loading="lazy"
            srcset="https://unsplash.it/800/800 800w,
https://unsplash.it/500/500 500w,
https://unsplash.it/300/300 300w,
https://unsplash.it/200/200 200w"
            title="An image title"
          />
          <header
            class="emotion-9 emotion-10"
          >
            <h1
              class="MuiTypography-root MuiTypography-h5 emotion-11"
            >
              Teambesprechung vom 23.05.
            </h1>
          </header>
          <p
            class="MuiTypography-root MuiTypography-h6 emotion-12"
          >
            Lead
          </p>
        </div>
      </a>
    </article>
    <article
      class="emotion-13 emotion-3"
      x="3"
      y="0"
    >
      <a
        class="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineNone emotion-4"
        href="https://example.com"
      >
        <div
          class="emotion-5 emotion-6"
        >
          <img
            alt="An image description"
            class="emotion-7 emotion-8"
            loading="lazy"
            srcset="https://unsplash.it/800/800 800w,
https://unsplash.it/500/500 500w,
https://unsplash.it/300/300 300w,
https://unsplash.it/200/200 200w"
            title="An image title"
          />
          <time
            class="emotion-20 emotion-21"
            datetime="2023-01-01T00:00:00.000Z"
          >
            01.01.2023
          </time>
          <header
            class="emotion-9 emotion-10"
          >
            <h1
              class="MuiTypography-root MuiTypography-h5 emotion-11"
            >
              Pre Title: Title
            </h1>
          </header>
          <p
            class="MuiTypography-root MuiTypography-h6 emotion-12"
          >
            Lead
          </p>
        </div>
      </a>
    </article>
    <article
      class="emotion-26 emotion-3"
      x="7"
      y="0"
    >
      <a
        class="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineNone emotion-4"
        href="https://example.com"
      >
        <div
          class="emotion-5 emotion-6"
        >
          <img
            alt="An image description"
            class="emotion-7 emotion-8"
            loading="lazy"
            srcset="https://unsplash.it/800/800 800w,
https://unsplash.it/500/500 500w,
https://unsplash.it/300/300 300w,
https://unsplash.it/200/200 200w"
            title="An image title"
          />
          <header
            class="emotion-9 emotion-10"
          >
            <h1
              class="MuiTypography-root MuiTypography-h5 emotion-11"
            >
              Pre Title: Title
            </h1>
          </header>
          <p
            class="MuiTypography-root MuiTypography-h6 emotion-12"
          >
            Lead
          </p>
        </div>
      </a>
    </article>
    <article
      class="emotion-37 emotion-3"
      x="0"
      y="4"
    >
      <a
        class="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineNone emotion-4"
        href="https://example.com"
      >
        <div
          class="emotion-5 emotion-6"
        >
          <img
            alt="An image description"
            class="emotion-7 emotion-8"
            loading="lazy"
            srcset="https://unsplash.it/800/800 800w,
https://unsplash.it/500/500 500w,
https://unsplash.it/300/300 300w,
https://unsplash.it/200/200 200w"
            title="An image title"
          />
          <time
            class="emotion-20 emotion-21"
            datetime="2023-01-01T00:00:00.000Z"
          >
            01.01.2023
          </time>
          <header
            class="emotion-9 emotion-10"
          >
            <h1
              class="MuiTypography-root MuiTypography-h5 emotion-11"
            >
              Pre Title: Title
            </h1>
          </header>
          <p
            class="MuiTypography-root MuiTypography-h6 emotion-12"
          >
            Lead
          </p>
        </div>
      </a>
    </article>
    <article
      class="emotion-50 emotion-3"
      x="3"
      y="6"
    >
      <a
        class="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineNone emotion-4"
        href="https://example.com"
      >
        <div
          class="emotion-5 emotion-6"
        >
          <img
            alt="An image description"
            class="emotion-7 emotion-8"
            loading="lazy"
            srcset="https://unsplash.it/800/800 800w,
https://unsplash.it/500/500 500w,
https://unsplash.it/300/300 300w,
https://unsplash.it/200/200 200w"
            title="An image title"
          />
          <header
            class="emotion-9 emotion-10"
          >
            <h1
              class="MuiTypography-root MuiTypography-h5 emotion-11"
            >
              Pre Title: Title
            </h1>
          </header>
          <p
            class="MuiTypography-root MuiTypography-h6 emotion-12"
          >
            Lead
          </p>
        </div>
      </a>
    </article>
    <article
      class="emotion-61 emotion-3"
      x="7"
      y="8"
    >
      <a
        class="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineNone emotion-4"
        href="https://example.com"
      >
        <div
          class="emotion-5 emotion-6"
        >
          <img
            alt="An image description"
            class="emotion-7 emotion-8"
            loading="lazy"
            srcset="https://unsplash.it/800/800 800w,
https://unsplash.it/500/500 500w,
https://unsplash.it/300/300 300w,
https://unsplash.it/200/200 200w"
            title="An image title"
          />
          <header
            class="emotion-9 emotion-10"
          >
            <h1
              class="MuiTypography-root MuiTypography-h5 emotion-11"
            >
              Teambesprechung vom 23.05.
            </h1>
          </header>
          <p
            class="MuiTypography-root MuiTypography-h6 emotion-12"
          >
            Lead
          </p>
        </div>
      </a>
    </article>
  </div>
</DocumentFragment>
`;

exports[`Teaser Grid Flex Block should render WithClassName 1`] = `
"Snapshot Diff:
- First value
+ Second value

@@ -1,8 +1,8 @@
  <DocumentFragment>
    <div
-     class="css-1iy4fdm-TeaserGridFlexBlockWrapper e1je8zv30"
+     class="extra-classname css-1iy4fdm-TeaserGridFlexBlockWrapper e1je8zv30"
    >
      <article
        class="css-8ks3rt-TeaserWrapper e18of27r4"
        x="0"
        y="0""
`;

exports[`Teaser Grid Flex Block should render WithEmotion 1`] = `
"Snapshot Diff:
Compared values have no visual difference."
`;
